argoverse/utils/sim2.py

```diff
@@ -141,3 +144,38 @@ def transform_from(self, point_cloud: np.ndarray) -> np.ndarray:
     def transform_point_cloud(self, point_cloud: np.ndarray) -> np.ndarray:
         """Alias for `transform_from()`, for synchrony w/ API provided by SE(2) and SE(3) classes."""
         return self.transform_from(point_cloud)
+
+    def save_as_json(self, save_fpath: Union[str, "os.PathLike[str]"]) -> None:
```

Comment:
How do you want to handle the typing of `Union[str, "os.PathLike[str]"]`? In some files we're doing this: https://github.com/argoai/argoverse-api/blob/9ad7251e863590603c3f6064a45830dabd3a15e2/argoverse/utils/pkl_utils.py#L8

Reply:
Thanks for the review @benjaminrwilson . I like the ` _PathLike = Union[str, "os.PathLike[str]"] ` macro. If you like it as well, I'll use it here too

Reply:
That sounds good. Is it recommended for us to repeat that macro in each file it's used?